### PR TITLE
Added simple custom error pages for the frontend proxy.

### LINKED ISSUE #270: Add custom HTTP error page to be served by the proxy frontend (eg: nginx)
The idea is to replace the standard error page like 
```
 502 Bad Gateway
------------------
  nginx v1.1.x
```
to a more pretty and meaningful error page, but still simple.

### PR DESCRIPTION
**Pull request by schaubl**
Added a python script to generate these error pages for different content-types.
Fix #270.
Corresponding update of the nginx configuration: slipstream/SlipStreamServerDeps#14
Related to slipstream/SlipStreamServer#310.

These error pages are small screen ready.

### CODE REVIEW

**Comment by rbf** [comment]
@schaubl 
Please see my comments on my previous commit 137ac2f.

**Comment by rbf** [comment]
FYI:

The error page curently renders like this:

screen shot 2015-04-07 at 12 25 09

With my suggestion above (137ac2f), it becomes:

screen shot 2015-04-07 at 12 25 24

**Comment by rbf** [comment]
:+1:
Cheers.
I'm merging...

File: clj/resources/static_content/errors/504.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <p><code>Error 504 - Gateway Time-out</code></p>
+            <h2>SlipStream didn't replied in time.<br/>It is probably overloaded.<br/>Please try again later.</h2>
```
Comment (konstan) [inline clj/resources/static_content/errors/504.html]: This needs to change: `didn't replied` to `didn't reply` in all the places of the commit.

Comment (schaubl) [inline clj/resources/static_content/errors/504.html]: Ok I will do that.

Comment (schaubl) [inline clj/resources/static_content/errors/504.html]: Fixed in e300204358fcf541859df7da84f6f86a82b8cbaf.

**Pull request merged**